packages/shared/src/components/squads/settings/SquadModerationSettingsSection.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement, useState } from 'react';
+import { SquadSettingsSection } from './SquadSettingsSection';
+import { Radio } from '../../fields/Radio';
+import { SourceMemberRole } from '../../../graphql/sources';
+import { Switch } from '../../fields/Switch';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { SimpleTooltip } from '../../tooltips';
+import { useToggle } from '../../../hooks/useToggle';
+
+interface SquadModerationSettingsSectionProps {
+  initialMemberPostingRole?: SourceMemberRole;
+  initialMemberInviteRole?: SourceMemberRole;
+  initialModerationRequired?: boolean;
+}
+
+const memberRoleOptions = [
+  {
+    label: 'All members (recommended)',
+    value: SourceMemberRole.Member,
+  },
+  {
+    label: 'Only moderators',
+    value: SourceMemberRole.Moderator,
+  },
+];
+
+export function SquadModerationSettingsSection({
+  initialMemberInviteRole,
+  initialMemberPostingRole,
+  initialModerationRequired,
+}: SquadModerationSettingsSectionProps): ReactElement {
+  const [isHoveringSwitch, setIsHoveringSwitch] = useState(false);
+  const [memberPostingRole, setMemberPostingRole] = useState(
+    () => initialMemberPostingRole || SourceMemberRole.Moderator,
+  );
+  const [memberInviteRole, setMemberInviteRole] = useState(
+    () => initialMemberInviteRole || SourceMemberRole.Member,
+  );
+  const [moderationRequired, setModerationRequired] = useToggle(
+    initialModerationRequired,
+  );
+
+  return (
+    <WidgetCard heading="🔒 Moderation Settings">
+      <div className="flex flex-col gap-6 px-4 py-2">
+        <SquadSettingsSection
+          title="Post content"
+          description="Choose who is allowed to post new content in this Squad."
+          className="flex"
+        >
+          <Radio
+            name="memberPostingRole"
+            options={memberRoleOptions}
+            value={memberPostingRole}
+            onChange={(value) => setMemberPostingRole(value)}
+          />
+        </SquadSettingsSection>
+        <SquadSettingsSection
+          title="Require post approval"
+          description="Turn this on to have admins or moderators approve every new post before it's published."
+        >
+          <SimpleTooltip
+            placement="top-start"
+            content={
+              <span className="p- max-w-[188px] p-2 text-center">
+                Only admins and moderators can post; their posts are
+                auto-published.
+              </span>
+            }
+            visible={
+              isHoveringSwitch &&
+              memberPostingRole === SourceMemberRole.Moderator
+            }
+          >
+            <span
+              className="max-w-fit cursor-pointer"
+              onMouseEnter={() => setIsHoveringSwitch(true)}
+              onMouseLeave={() => setIsHoveringSwitch(false)}
+            >
```

Comment:
Wrapped the switch in a span because disabled inputs do not fire events.

Reply:
I don't really see why this is needed?
The tooltip should just show automatically on hovering any type of element?

Reply:
Yes, if there is no condition set in the "visible" property, it will automatically show on hover on any type of element.

However, as mentioned, disabled inputs do not trigger events, so the "onMouseEnter" event is not being triggered by its child.

We also additionally only want to show the hover if moderation role is set to "moderator" _and_ the switch is disabled

Reply:
Not sure but in these cases maybe we want to suggest to design that it would be best if for these things we had any ℹ️ icon in the section that if user hovers on it, it shows the tooltip with the information. 
Rather then hack these things where it might not make too much sense 🤔

Reply:
> Not sure but in these cases maybe we want to suggest to design that it would be best if for these things we had any ℹ️ icon in the section that if user hovers on it, it shows the tooltip with the information. Rather then hack these things where it might not make too much sense 🤔

I agree. I did wonder if this would've been better off as a "hint" below the switch, but an info icon works well, too